### PR TITLE
Load Reporting hook

### PR DESCRIPTION
**Pull request by dgquintas**
Includes https://github.com/grpc/grpc/pull/6177

### CODE REVIEW

**Comment by dgquintas** [comment]
RFC

File: src/core/lib/channel/load_reporting_filter.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#include <string.h>
```
Comment (ctiller) [inline src/core/lib/channel/load_reporting_filter.c]: Please move this to src/core/ext/load_reporting and make it a plugin

**Comment by dgquintas** [comment]
?? I have no idea how this got closed. Reopening.

**Comment by ctiller** [comment]
What state is this in?

**Comment by dgquintas** [comment]
Ready for review (just merged with master).

**Comment by dgquintas** [comment]
I missed you comment about making it a plugin. Doing that now. Merged with master otherwise.

**Comment by ctiller** [comment]
It might need a remerge...

**Comment by dgquintas** [comment]
Ack. I'm wrapping up the filter/plugin changes as recently discussed.

**Comment by ctiller** [comment]
Can you resolve this?

**Comment by dgquintas** [comment]
Done.

**Comment by ctiller** [comment]
LGTM, merge on green

**Comment by dgquintas** [comment]
Flakes are known. Tagging for merging.

**Pull request merged**